README.md

```diff
@@ -1,98 +1,168 @@
 ```
 
-## Running the snapshotter
+## Configuration file
 
-To run the snapshotter a configuration file is required. This file is a YAML file with the following structure:
+### Passing the configuration file
 
-```yaml
-prefix: "prefix/path/to/store/the/backup"
-out: "/path/to/backup.tar"
-volumes:
-  - "/path/to/volume1"
-  - "/path/to/volume2"
+The snapshotter process requires the configuration file to be located at the `/config.yml` path inside the container. Therefore, the recommended way to pass the configuration is by mounting the following volume:
+
+```text
+--volume <path-to-config>:/config.yml
 ```
 
-The snapshotter tries to append the volumes to the backup tar file. If the tar file does not exists, or is empty, an error is raised. To initialize a proper empty tar file run the following command:
+Replace `<path-to-config>` with the absolute path to the configuration file on the host machine.
 
-```bash
-tar -cvf backup.tar --files-from /dev/null
-```
+### Configuration format
 
-You can run the snapshotter different times targeting the same output tar file using different prefix paths. This is useful to create backups of different containers in the same tar file.
+The snapshotter does not need too many configurations, only two options are necessary:
+
+1. `prefix`: is the prefix path to store the volumes inside the backup tarball file
+2. `volumes`: list of volume targets inside the container, should be absolute paths to a directory or a file inside the container
 
 ### Example
 
-Add the following configuration file to the root of this repository as `config.yml`:
+The following example configures the snapshotter to save volumes under the `prefix/path` inside the `backup.tar` file. Volumes could be directories like `volume1` and `volume3` or files like `volume2.txt`.
 
 ```yaml
-prefix: volumes/mock-avs-second
-out: /backup.tar
+prefix: "prefix/path"
 volumes:
-  - /cli
-  - /README.md
+ - /path/to/volume1
+ - /path/to/volume2.txt
+ - /path/to/volume3
 ```
 
-Run following command in the root of this repository:
+## Backup file
 
-```bash
-docker run --rm \
-    -v $(pwd)/backup.tar:/backup.tar \       # Bind the backup tar
-    -v $(pwd)/config.yml:/snapshotter.yml \  # Bind the configuration file
-    -v $(pwd)/cli:/cli \                     # Bind a volume to backup
-    -v $(pwd)/README.md:/README.md \         # Bind a file to backup
-    docker-volumes-snapshotter
-```
+### Initialize `tar` file
 
-Checking the content of the tar file:
+The backup tar file should be empty or not. However, in any case, it should end with 1024 zero bytes. These zero bytes consist of two empty blocks of 512 bytes each, which define an end-of-archive, as specified in [the spec](https://www.gnu.org/software/tar/manual/html_node/Standard.html). This requirement is necessary to enable the appending of files to existing tar files that contain volumes from other containers, but are located at different paths within the tarball.
```

Comment:
> "The backup tar file should be empty or not."

What do you mean by that?

Reply:
I replace `should` for `could` in this commit https://github.com/NethermindEth/docker-volumes-snapshotter/pull/1/commits/89ee98b0715b976c409e5f52a1a8b5d9ae959511. I think this clarifies the sentence.